tekton/utils_test.go

```diff
@@ -88,14 +94,14 @@ var _ = Describe("Utils", func() {
 
 	It("can get git-url", func() {
 		git_url, _ := tekton.GetComponentSourceGitUrl(pipelineRun)
-		if git_url != "https://github.com/devfile-samples/devfile-sample-java-springboot-basic" {
-			Fail(fmt.Sprintf("Expected git_url is https://github.com/devfile-samples/devfile-sample-java-springboot-basic, but got %s", git_url))
+		if git_url != "https://github.com/upstream-user/devfile-sample-java-springboot-basic" {
+			Fail(fmt.Sprintf("Expected git_url is https://github.com/upstream-user/devfile-sample-java-springboot-basic, but got %s", git_url))
 		}
 		klog.Infoln("Got expected git_url")
 	})
 
 	It("can return err when can't find result for CHAINS-GIT_URL", func() {
```

Comment:
I think this description should be updated to say that we can't find the `git-url` pipeline parameter.